src/main/scala/com/codacy/client/bitbucket/v2/service/AuthorizationServices.scala

```diff
@@ -7,17 +7,17 @@ import com.codacy.client.bitbucket.v2.Authorization.{RefreshCredentials, Refresh
 
 class AuthorizationServices(client: BitbucketClient) {
 
+  private val authorizationUrl = "https://bitbucket.org/site/oauth2/access_token"
```

Comment:
Codacy found an issue: [authorizationUrl should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)